build.gradle

```diff
@@ -53,9 +53,23 @@ subprojects {
   description = "Elasticsearch subproject ${project.path}"
 }
 
+apply plugin: 'nebula.info-scm'
+String licenseCommit
```

Comment:
with this code moved, do we still need it in x-pack/build.gradle?

Reply:
Woops, yeah, let me get that all the way out.